specification/app/data-plane/readme.md

```diff
@@ -0,0 +1,51 @@
+# Container Apps Data Plane Api
+
+> see https://aka.ms/autorest
+This is the AutoRest configuration file for Container Apps Data Plane Api.
+
+## Getting Started
+
+To build the SDKs for My API, simply install AutoRest via `npm` (`npm install -g autorest`) and then run:
+
+> `autorest readme.md`
+To see additional help and options, run:
+
+> `autorest --help`
+For other options on installation see [Installing AutoRest](https://aka.ms/autorest/install) on the AutoRest github page.
+
+---
+
+## Configuration
+
+### Basic Information
+
+These are the global settings for the Container Apps Data Plane Api.
+
+```yaml
+openapi-type: data-plane
+tag: package-2024-09-01
+```
+### Tag: package-2024-09-01
+These settings apply only when `--tag=package-2024-09-01` is specified on the command line.
+
+```yaml $(tag) == 'package-2024-09-01'
+input-file:
+  - Microsoft.App/stable/2024-09-01/containerapps.json
```

Comment:
The Avocado build check is complaining about this line. Please investigate and fix.

Reply:
Thanks, fixed